assets/js/components/ChecksCatalog/ChecksCatalog.jsx

```diff
@@ -8,20 +8,22 @@ import { getCatalog } from '@state/selectors/catalog';
 import { updateCatalog } from '@state/actions/catalog';
 import {
   providerData,
-  getLabels,
-  getProviderByLabel,
+  checkProviderExists,
 } from '@components/ProviderLabel/ProviderLabel';
 import PageHeader from '@components/PageHeader';
 import CatalogContainer from './CatalogContainer';
 import CheckItem from './CheckItem';
 import ProviderSelection from './ProviderSelection';
 
-const ALL_FILTER = 'All';
+const ALL_FILTER = 'all';
+const ALL_FILTER_TEXT = 'All';
 const updatedProvider = {
-  default: { label: ALL_FILTER },
+  [ALL_FILTER]: { label: ALL_FILTER_TEXT },
```

Comment:
This extended providerData (which also contains the `all`) could also make sense to be close to where we define the available providers in `@components/ProviderLabel/ProviderLabel` or even in `ProviderSelection`.

Not a biggie though.

Reply:
Alright, let's keep it as it is :+1: